src/discord-client.c

```diff
@@ -234,6 +249,7 @@ discord_cleanup(struct discord *client)
         free(client->workers);
     }
     free(client);
+    ccord_global_cleanup();
```

Comment:
If we are keeping this, you want to move it inside the `is_original == true` condition. The idea of `discord_clone()` is allowing creating many `struct discord` which share resources with the original client, in a parallel context, so we must assume the original will be cleaned up last.